examples/winit.rs

```diff
@@ -23,11 +25,14 @@ fn main() {
             } => {
                 *control_flow = ControlFlow::Exit;
             }
+            winit::event::Event::MainEventsCleared => {
+                window.request_redraw();
+            }
             winit::event::Event::RedrawRequested(_) => {
                 context.make_current();
 
                 unsafe {
-                    gl::ClearColor(1.0, 0.0, 1.0, 1.0);
```

Comment:
Adds some per-frame change, makes it easier to tell if things are still working after a window resize